src/metabase/api/card.clj

```diff
@@ -692,4 +693,14 @@
   [:as {query :body}]
   (related/related (query/adhoc-query query)))
 
+(api/defendpoint ^:streaming POST "/pivot/:card-id/query"
+  "Run the query associated with a Card."
+  [card-id :as {{:keys [parameters ignore_cache]
+                 :or   {ignore_cache false}} :body}]
+  {ignore_cache (s/maybe s/Bool)}
+
```

Comment:
https://guide.clojure.style/#no-blank-lines-within-def-forms

Reply:
I'll take the blank lines out, but I also want to go on the record that this stylistic choice goes against readability. Blank lines exist for readability, and to separate logical sections of functions.

Reply:
I agree RE readability, but I think it's just too weird for us to adopt a convention like "use newlines to break up sections inside top-level forms" when 99% of the Lisp code out in the wild doesn't do it that way.

I like to use comment lines to break big forms up a bit into separate sections. Besides visually breaking things up, similarly to blank lines, they have the added bonus of explaining what the next logical section is doing